style.css

```diff
@@ -443,7 +443,175 @@ textarea {
   color: #509467;
 }
 
+/** pop up **/
+.my-works-details {
+  display: grid;
+  padding: 16px;
+  font-size: 16px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  border-radius: 16px;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  height: 100%;
+  width: 100%;
+  z-index: 2;
+  transition: 2000ms ease-in-out;
+  transform: translate(-50%, -50%) scale(1);
+}
+
+.btn-close-pop-up {
+  position: absolute;
+  top: 0%;
+  right: 5%;
+  background: none;
+}
+
+.work-image-details {
+  overflow: hidden;
+  box-sizing: border-box;
+  border-radius: 16px;
+}
+
+.work-image-details > img {
+  width: 100%;
+  min-height: 220px;
+  object-fit: cover;
+  content: url("./img/background/bk.png");
+}
+
+.work-title-mobile {
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 1.5em;
+  color: #172b4d;
+  text-align: left;
+}
+
+.work-tags-mobile {
+  list-style: none;
+  padding: 0;
+  display: flex;
+  justify-content: flex-start;
+}
+
+.tag-mobile {
+  background: #ebf0ee;
+  color: #3a4a42;
+  padding: 0.5em 0.75em 0.5em;
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 0.75em;
+  line-height: 1em;
+  letter-spacing: 0.03em;
+  border-radius: 0.25em;
+}
+
+.work-discription-mobile {
+  color: #344563;
+  text-align: left;
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 1em;
+  line-height: 1.5em;
+  padding-bottom: 15px;
+}
+
+.work-button-mobile {
+  display: flex;
+  justify-content: space-between;
+}
+
+.work-detail-btn-mobile {
+  display: flex;
+  margin-top: 1.5em;
+  margin-bottom: 1.5em;
+  border: none;
+  width: fit-content;
+  align-self: center;
+  background: #0e5a3b;
+  font-weight: 500;
+  font-size: 16px;
+  letter-spacing: 2px;
+  padding: 1em 1.5em;
+  color: white;
+  border-radius: 3px;
+  text-decoration: none;
+}
+
+.work-detail-btn-mobile > img {
+  margin-left: 1.3em;
+}
```

Comment:
- [ ] You did an amazing job styling the mobile popup :+1:, but I think it would be better if you add the required background as the Figma template. It means you should have a card in the middle of your page and a blurred background.

- [ ] Kindly make the title bigger just like the Figma :+1: 

- [ ] Also, you should make the buttons a little bit smaller to match as much as possible with the template :+1: 

**Your design**

![Screenshot from 2023-02-03 06-16-30](https://user-images.githubusercontent.com/105117832/216591534-85bc640e-0ab1-4849-80f2-fec70b19e8d4.png)

**Figma**

![Screenshot from 2023-02-03 06-20-58](https://user-images.githubusercontent.com/105117832/216591580-9389269b-a7a1-439f-a87b-b60ce8ff8ac3.png)